app/services/api/v3/finance/statements/index.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Api
+  module V3
+    module Finance
+      module Statements
+        class Index
+          attr_reader :cpd_lead_provider, :params
+
+          def initialize(cpd_lead_provider:, params:)
+            @cpd_lead_provider = cpd_lead_provider
+            @params = params
+          end
+
+          def statements
+            statement_class
+              .includes(:cohort)
+              .where(
+                cpd_lead_provider:,
+                cohort_id: with_cohorts.map(&:id),
+              ).order(created_at: :asc)
+          end
+
+        private
+
+          def filter
+            params[:filter] ||= {}
+          end
+
+          def with_cohorts
+            return Cohort.where(start_year: filter[:cohort].split(",")) if filter[:cohort].present?
+
+            Cohort.where("start_year > 2020")
+          end
+
+          def statement_class
```

Comment:
Suggestion: inject this value at runtime rather than trying to guess what is the correct one in this class, gives flexibility to lookup `Finance::Statement::ECF::Paid`, `Finance::Statement::ECF::Payable` etc..

Reply:
addressed in https://github.com/DFE-Digital/early-careers-framework/pull/2564/commits/4cbecd97524261228f5dcd155ff0986cc6392314